src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball extends ColorSupplier {
+    private String color;
+    private int number;
+
+    public Ball() {
```

Comment:
Don't generate values inside the constructor - you should pass them as constructor arguments